kibbeh/src/ui/icons/SearchIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+interface SearchIconProps extends React.ComponentPropsWithoutRef<"svg"> {
+  className?: string;
+}
+
+export const SearchIcon: React.FC<SearchIconProps> = (props) => {
```

Comment:
The svg has its own size already set. Try to use it like this https://github.com/benawad/dogehouse/blob/40f987073f21eb252d5c4258a919d486bf47712f/kibbeh/src/ui/SearchHistory.tsx#L3 so you don't run into sizing problems and also is consistent with the other search components.

Reply:
I see new icons have been added. I replaced what I had previously with the existing `SmSolidSearch`. I had to change its stroke color so that we can override it from the parent component.